lib/testLauncher/repl.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Suitest interactive REPL
+ */
+
+const repl = require('repl');
+
+let isActive = false;
+
+/**
+ * Start REPL
+ * @param {Object} suitest instance
+ * @returns {Promise<void>}
+ */
+function startRepl(suitest) {
+	const replInstance = repl.start({
+		prompt: '> ',
+	});
+
+	isActive = true;
+	replInstance.context.suitest = suitest;
+
+	return new Promise(resolve => {
+		suitest.resume = () => {
```

Comment:
`resume` method available only after calling `suitest.interactive()`?

Reply:
yes, only in repl

Reply:
I just realized, user is also able to exit repl with `.exit` command, 
https://nodejs.org/api/repl.html#repl_commands_and_special_keys
I'll investigate, suitest.resume could be obsolete.

Reply:
If `.exit` would only stop REPL and normal execution would resume - sure. But if it's exiting the process - we need `suitest.resume()` still.
There is an option to add your own syntax for special commands I think. Would also be an option.

Reply:
.exit doesn't close process